<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
QWidget{
    background: #3A3B3C;
}

QPushButton{
	background-color:#5b5b5b;
	color:#bcbcbc;
}

QProgressBar{
	color:#bcbcbc;
}

QLabel{
	color:#bcbcbc;
}

/*---------------------------  QTreeWidget --------------------------*/
QTreeWidget{
    color : rgb(236, 236, 236);
    font-size: 10pt;
    font-weight: bold;
}

QTreeWidget::item{
    margin:2px;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_content">
        <property name="font">
         <font>
          <pointsize>15</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>ShotInfo</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTreeWidget" name="tree_shot">
        <attribute name="headerVisible">
         <bool>true</bool>
        </attribute>
        <attribute name="headerCascadingSectionResizes">
         <bool>false</bool>
        </attribute>
        <attribute name="headerDefaultSectionSize">
         <number>90</number>
        </attribute>
        <column>
         <property name="text">
          <string>shot</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>mov</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>hip</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>nk</string>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <widget class="QProgressBar" name="progressBar">
        <property name="value">
         <number>0</number>
        </property>
        <property name="textVisible">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_run">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>RUN</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
